.gitignore

```
@@ -28,6 +28,7 @@ tokenizer.json
 config.json
 generation_config.json
 vocabulary.json
+packages/*/.model
```

We should just ignore every `.model/` directory.
```suggestion
.model/
```

---

changed :+1: